src/Uno.UI.Toolkit/Storage/StorageFileHelper.cs

```diff
@@ -0,0 +1,43 @@
+﻿#nullable enable
+using System.Threading;
+using System;
+using System.Threading.Tasks;
+using Uno;
+using System.Reflection;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Uno.UI.Toolkit;
+
+public partial class StorageFileHelper
+{
+	/// <summary>
+	/// Determines if an asset or resource exists within application package
+	/// </summary>
+	/// <param name="fileName">relative file path</param>
+	/// <returns>A task that will complete with a result of true if file exists, otherwise with a result of false.</returns>
+	public static async Task<bool> ExistsInPackage(string fileName) => await FileExistsInPackage(fileName);
+
+#if NET461 || __NETSTD_REFERENCE__
+	private static Task<bool> FileExistsInPackage(string fileName)
+		=> throw new NotImplementedException();
+#endif
+
+#if __SKIA || WINDOWS || WINDOWS_UWP || WINUI
```

Comment:
Sorry, that was a typo in my comment.

```suggestion
#if __SKIA__ || WINDOWS || WINDOWS_UWP || WINUI
```